core/src/main/res/values/strings.xml

```diff
@@ -624,118 +624,118 @@
     <string name="pref_resumeAfterCall_title">Resume after Call</string>
-    <string name="pref_restart_required">AntennaPod has to be restarted for this change to take effect.</string>
+    <string name="pref_restart_required">Restart the app to use the new settings.</string>
 
     <!-- Online feed view -->
     <string name="subscribe_label">Subscribe</string>
-    <string name="subscribing_label">Subscribing&#8230;</string>
+    <string name="subscribing_label">Subscribing…</string>
     <string name="preview_episode">Preview</string>
     <string name="stop_preview">Stop preview</string>
 
     <!-- Content descriptions for image buttons -->
     <string name="rewind_label">Rewind</string>
     <string name="fast_forward_label">Fast forward</string>
-    <string name="increase_speed">Increase speed</string>
-    <string name="decrease_speed">Decrease speed</string>
+    <string name="increase_speed">Faster</string>
+    <string name="decrease_speed">Slower</string>
     <string name="media_type_video_label">Video</string>
-    <string name="status_downloading_label">Episode is being downloaded</string>
-    <string name="in_queue_label">Episode is in the queue</string>
-    <string name="is_favorite_label">Episode is marked as favorite</string>
-    <string name="load_next_page_label">Load next page</string>
+    <string name="status_downloading_label">Downloading episode…</string>
+    <string name="in_queue_label">The episode is enqueued</string>
+    <string name="is_favorite_label">Favourited episode</string>
+    <string name="load_next_page_label">Next page…</string>
     <string name="position">Position: %1$s</string>
     <string name="apply_action">Apply action</string>
     <string name="play_chapter">Play chapter</string>
     <string name="prev_chapter">Previous chapter</string>
     <string name="next_chapter">Next chapter</string>
 
     <!-- Feed settings/information screen -->
-    <string name="authentication_label">Authentication</string>
+    <string name="authentication_label">Log in</string>
     <string name="authentication_descr">Change your username and password for this podcast and its episodes.</string>
     <string name="feed_tags_label">Tags</string>
-    <string name="feed_tags_summary">Change the tags of this podcast to help organize your subscriptions</string>
+    <string name="feed_tags_summary">Tag your subscriptions to organize them.
     <string name="feed_folders_include_root">Show in main list</string>
-    <string name="auto_download_settings_label">Auto Download Settings</string>
+    <string name="auto_download_settings_label">Auto-downloading</string>
     <string name="episode_filters_label">Episode Filter</string>
-    <string name="episode_filters_description">List of terms used to decide if an episode should be included or excluded when auto downloading</string>
+    <string name="episode_filters_description">Terms to prevent or auto-download episodes</string>
```

Comment:
I think this is harder to understand now

Reply:
Does "Terms to" need to be there?